doc/release-notes/rl-2511.section.md

```diff
@@ -9,7 +9,7 @@
 
 <!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
 
-- Create the first release note entry in this section!
+- The `boot.readOnlyNixStore` has been removed. Control over bind mount options on `/nix/store` is now offered by the `boot.nixStoreMountOpts` option.
```

Comment:
This entry would have belonged in `nixos/doc/release-notes`, I think. Note that it got lost in [a merge](https://github.com/NixOS/nixpkgs/commit/dfaa06841267553648643850c561f61cf78aa10d#diff-300d64b8febbf8f80bf778114bd0b70a2b31705d602365a32f7b5a2857764090) anyways. cc @K900

Reply:
No, I moved it to the right location in that merge.

Reply:
Good, must've checked on the wrong branch.